src/views/App/App.js

```diff
@@ -22,6 +22,7 @@ class App extends Component {
           <Route path={HOME_URL} component={MainScreen} />
           <Route path={CONTENT_URL} component={ContentScreen} />
           <Route path={ABOUT_URL} component={AboutScreen} />
+          <Route component={MainScreen} />
```

Comment:
Probably a better way but the cordova has trouble with routing.  This hack is basically serving the mainScreen as the 404 page and from there the routing works as expected. ie. no white screen. 👍

Reply:
Should make this render the splash screen as that has the redirect if they've already been through them.

Reply:
Should make this render the splash screen by default as it has the redirect if they've already viewed them.

[Refer to this](https://github.com/drewdiddy611/yffr-app/blob/546ba27f53003037321472ebfffb195eff0086bb/src/views/pages/SplashScreen/SplashScreen.js#L16)